docs/changelog.txt

```diff
@@ -36,6 +36,9 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Documentation
 - Added more client library implementations to the `remote interface docs <remote-client-libs>`
 
+## Internals
+- Lua test files are no longer required to start with standard script documentation (since they aren't runnable scripts)
```

Comment:
The problematic files didn't exist before, so this doesn't need to be mentioned. (Any tests in the DFHack repo were exempt from this, and they were the only ones that existed previously.)